src/plugin/bootstrap/InitialisePlugin.cpp

```diff
@@ -168,6 +169,9 @@ namespace UKControllerPlugin {
             LogInfo("Plugin loaded as primary instance");
         }
 
+        LogInfo("Checking UKCPa is up to date");

```

Comment:
```suggestion
        LogInfo("Checking UK Controller Pack is up to date");

```

We aren't limited by memory these days 😄